components/helper/ShareButtons.tsx

```diff
@@ -9,9 +11,132 @@ import {
   TwitterShareButton,
 } from 'react-share';
 
-export default function ShareButtons() {
+type ShareButtonsProps = {
+  popup: boolean;
+  setCopied: React.Dispatch<React.SetStateAction<boolean>>;
+  copied: boolean;
+};
+
+export default function ShareButtons({
+  popup, setCopied, copied,
+}: ShareButtonsProps) {
   const url = 'https://clearviction.org/calculator/head-initial-1-cont';
 
+  const style = {
+    fontWeight: 400,
+    lineHeight: '1.375rem',
+    color: '#2F3554',
+    textDecoration: 'underline',
+    marginLeft: '.4375rem',
+  };
+
+  const iconsStyle = { display: 'flex', alignItems: 'center', cursor: 'pointer' };
+
+  const handleCopyClick = async () => {
+    await navigator.clipboard.writeText(url);
+    setCopied(true);
+  };
+
+  function handleEmailClick() {
+    const body = encodeURIComponent(url);
+    const mailtoUrl = `mailto:?body=${body}`;
+    window.open(mailtoUrl);
+  }
+
+  if (copied) {
+    return (
+      <Box
+        className="copied-message-box"
+        sx={{
+          width: '20.3125rem',
+          height: '5.9375rem',
+          padding: '1rem',
+          borderRadius: '6px',
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+        }}
+      >
+        <Typography
+          variant="h6"
+          sx={{
+            marginBottom: '0',
+          }}
+        >
+          Link copied!
+        </Typography>
+      </Box>
+    );
+  }
+
+  if (popup) {
+    return (
+      <Box
+        className="popup-main-container-box"
+        sx={{
+          width: '28.1rem', height: '20.625rem', padding: '1rem', borderRadius: '6px',
+        }}
+      >
+
+        <Box
+          className="popup-inner-container-title-box"
+          sx={{
+            display: 'flex', flexDirection: 'column', marginLeft: '1rem', gap: '.0094rem',
+          }}
+        >
+
+          <Typography
+            variant="body1"
+            sx={{
+              fontSize: {
+                xs: '1.3rem',
```

Comment:
One of the benefits of using rem for sizing, is that it is responsive and scales itself, so you only ever need to declare one font size. `body1` is `1.25 rem`, so I think we can just delete the font size here all together.

Reply:
That makes sense but is 1.25 rem not to small if we that is comparing to 1.5 rem which is the size for sm?

Reply:
I'm not sure what you mean.

Reply:
So if the default for body1 is 1.25 rem would that not be noticeably smaller than the font value I had put for dextop sizes: 1.5rem?